# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/psmisc/gcc33.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/killall.c.orig	2003-05-20 14:36:30.000000000 -0400
+++ ./src/killall.c	2003-05-20 14:37:15.000000000 -0400
@@ -395,8 +395,7 @@
 #ifdef FLASK_LINUX
   fprintf (stderr, "  -S,--Sid            kill only process(es) having sid\n");
   fprintf (stderr, "  -c,--context        kill only process(es) having scontext\n");
-  fprintf(stderr, "   (-s, -c are mutually exclusive and must precede other
-arguments)\n\n");
+  fprintf(stderr, "   (-s, -c are mutually exclusive and must precede other arguments)\n\n");
 #endif /*FLASK_LINUX*/
 }
 
